simple_static_website/main.tf

```
@@ -54,6 +54,13 @@ resource "random_string" "suffix" {
 resource "aws_s3_bucket" "this" {
   bucket        = var.s3_bucket_name == "" ? "${var.domain_name_source}-${random_string.suffix.result}" : var.s3_bucket_name
   force_destroy = var.force_destroy_s3_bucket
+
+  lifecycle {
+    ignore_changes = [
+      tags["version"],
+      tags_all["version"]
```

Might be a dumb question, but do these need to be like so since the tab is uppercase `Version`?
```suggestion
      tags["Version"],
      tags_all["Version"]
```

---

You're the hero we need